cmd/translate/main_grpc_integration_test.go

```diff
@@ -684,33 +686,43 @@ func Test_UpdateTranslation_gRPC(t *testing.T) {
 	createTranslation(ctx, t, service.GetId(), &translatev1.Translation{Original: true, Language: langs[0].String()})
 	createTranslation(ctx, t, service.GetId(), &translatev1.Translation{Original: false, Language: langs[1].String()})
 
-	happyReq := randUpdateTranslationReq(t, service.GetId(), &translatev1.Translation{Language: langs[1].String()})
+	happyReq := randUpdateTranslationReq(t, service.GetId(), &translatev1.Translation{Language: langs[1].String()}, nil)
+
+	req := randUpdateTranslationReq(t, service.GetId(), &translatev1.Translation{
+		Original: true, Language: langs[0].String(),
+	},
+		&field_mask.FieldMask{Paths: []string{"original"}})
 
 	// different language without translation
 	notFoundTranslationReq := randUpdateTranslationReq(t,
-		service.GetId(), &translatev1.Translation{Language: langs[2].String()})
+		service.GetId(), &translatev1.Translation{Language: langs[2].String()}, nil)
 
 	notFoundServiceID := randUpdateTranslationReq(t,
-		gofakeit.UUID(), &translatev1.Translation{Language: langs[1].String()})
+		gofakeit.UUID(), &translatev1.Translation{Language: langs[1].String()}, nil)
 
 	invalidArgumentNilTranslationReq := &translatev1.UpdateTranslationRequest{ServiceId: service.GetId()}
 
-	invalidArgumentUndTranslationLanguageReq := randUpdateTranslationReq(t, gofakeit.UUID(), nil)
+	invalidArgumentUndTranslationLanguageReq := randUpdateTranslationReq(t, gofakeit.UUID(), nil, nil)
 	invalidArgumentUndTranslationLanguageReq.Translation.Language = ""
 
 	originalAlreadyExistsReq := randUpdateTranslationReq(t,
-		service.GetId(), &translatev1.Translation{Language: langs[1].String(), Original: true})
+		service.GetId(), &translatev1.Translation{Language: langs[1].String(), Original: true}, nil)
 
 	tests := []struct {
 		request      *translatev1.UpdateTranslationRequest
 		name         string
 		expectedCode codes.Code
 	}{
 		{
-			name:         "Happy Path",
+			name:         "Happy Path update all",
 			request:      happyReq,
 			expectedCode: codes.OK,
 		},
+		{
+			name:         "Happy path update original field",
```

Comment:
update_mask 'original'? What about update_mask targeting 'messages' field?

Reply:
does not matter, as this test do not check, if implementation doing the right thing, it just checks that request gone through and returned ok code.

tho It could test, not only the status code, but also the returned value